### PR TITLE
Add warning panel

### LINKED ISSUE #659: Warning panel style
Reintroduce a 'warning' panel when the consequences of the users action could have a negative effect.  e.g. The EQ 'remove person' confirmation message: 

Screenshot 2019-11-08 at 15 49 13

_Originally posted by @jrbarnes9 in https://github.com/ONSdigital/design-system/issues/49#issuecomment-551880318_

### PR DESCRIPTION
**Pull request by rmccar**
PR to add a warning version of the panel component. Fixes #659

### CODE REVIEW

**Comment by rmccar** [comment]
[Issue #659: Warning panel style](https://trello.com/c/FPC5Tirr/453-issue-659-warning-panel-style)

File: src/components/panel/_panel.scss [hunk 1]
```diff
@@ -70,9 +70,30 @@
       padding: 2rem;
     }
   }
+
+  &--warn & {
+    &__icon {
+      color: #fff;
+      font-weight: 900;
+      display: inline-block;
```
Comment (boxadesign) [inline src/components/panel/_panel.scss]: Redundant when using `position: absolute`

File: src/components/panel/_panel.scss [hunk 2]
```diff
@@ -70,9 +70,30 @@
...
+      position: absolute;
+      left: 0;
```
Comment (boxadesign) [inline src/components/panel/_panel.scss]: Will also need a `top:` value

File: src/components/panel/_panel.scss [hunk 3]
```diff
@@ -70,9 +70,30 @@
...
+      min-width: 2rem;
+      min-height: 2rem;
+      margin-top: -0.2rem;
```
Comment (boxadesign) [inline src/components/panel/_panel.scss]: Not required when `top:` is added

File: src/components/panel/_panel.scss [hunk 4]
```diff
@@ -70,9 +70,30 @@
...
+      margin-left: 0.5rem;
```
Comment (boxadesign) [inline src/components/panel/_panel.scss]: This can be done with the `left:` value

**Comment by boxadesign** [comment]
@rmccar The icon has absolute positioned but the parent panel needs to be position relative otherwise the icon will find the closest ancestor with relative positioning and will set itself to that. 

By adding `position: relative` to the `panel` you'll probably see some layout issues. The title will need some padding that takes into account the width of the icon.

**Pull request merged**